mindmaps-graph/src/main/java/io/mindmaps/graph/internal/AbstractMindmapsGraph.java

```diff
@@ -186,9 +187,11 @@ public ConceptImpl getConcept(Schema.ConceptProperty key, String value) {
     }
 
     public Set<String> getModifiedCastingIds(){
-        Set<String> relationIds = new HashSet<>();
-        getConceptLog().getModifiedCastings().forEach(c -> relationIds.add(c.getId()));
-        return relationIds;
+        return getConceptLog().getModifiedCastingIds();
+    }
+
+    public Set<String> getModifiedResourceIds(){
+        return getConceptLog().getModifiedResourceIds();
```

Comment:
Perhaps you don't need to expose these methods. Instead, just `getConceptLog()` below